app/code/Magento/ImportExport/Model/Import/Entity/AbstractEntity.php

```diff
@@ -408,8 +416,8 @@ protected function _saveValidatedBunches()
             if ($source->valid()) {
                 try {
                     $rowData = $source->current();
-                    if (array_key_exists('sku', $rowData)) {
-                        $skuSet[$rowData['sku']] = true;
+                    if ($this->uniqueField !== null) {
```

Comment:
Please, revert array_key_exists check. We'll have undefined index error if $rowData does not contain $this->uniqueField field

Reply:
ok